subiquity/server/controllers/filesystem.py

```diff
@@ -294,6 +294,18 @@ def load_autoinstall_data(self, data):
         # log.debug("self.ai_data = %s", data)
         self.ai_data = data
 
+    # The identity and user-data section are optional if we are only installing
+    # the reset partition, however the identity controller needs to know this
+    # before the filesystem controller naturally sets this. So this is a
+    # temporary function to inspect the outcome.
+    # See: https://github.com/canonical/subiquity/pull/1965
+    def _reset_parition_only(self):
+        # Cheating and checking the server's config makes this function
```

Comment:
I think this isn't cheating.  Filesystem is answering filesystem questions.

Reply:
Ah fair, thanks. It was "cheating" in the identity controller.